src/Functions/GeoHash.cpp

```diff
@@ -115,6 +115,7 @@ inline Encoded merge(const Encoded & encodedLon, const Encoded & encodedLat, uin
     result.fill(0);
 
     const auto bits = (precision * BITS_PER_SYMBOL) / 2;
+    assert(bits <= 255);
```

Comment:
This is to guide static analyzer. Actually it shouldn't overflow with the given range of precision.